### PR TITLE
[Feature] End User Vault Refresh

### PR DESCRIPTION
**Pull request by differsthecat**
https://bitwarden.atlassian.net/jira/software/projects/SG/boards/34?selectedIssue=SG-221
https://bitwarden.atlassian.net/jira/software/projects/SG/boards/34?selectedIssue=SG-222

## Type of change

- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Merge End User Vault Refresh changes into master:

* https://bitwarden.atlassian.net/browse/SG-78 (Browser)
* https://bitwarden.atlassian.net/jira/software/projects/SG/boards/34?selectedIssue=SG-80 (Desktop)

Original PRs to feature branch:
* https://github.com/bitwarden/bitwarden/pull/2463 (Browser)
* https://github.com/bitwarden/desktop/pull/1487 (Desktop)

Depends on https://github.com/bitwarden/jslib/pull/790

## Testing requirements

Please make sure all filters, folders, collections everything load correctly and respond to the new org filter correctly. 
Also, please thoroughly test vaults that do not belong to organizations. Most users will not have any orgs and I want to make sure there are no unwanted side effects to most users because of these changes.

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [x] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by eliykat** [comment]
Reviewing currently:
* browser changes: approved ✅  - all are from PR #2463 and were approved - no unexpected changes generally
* desktop changes: in progress

Also waiting on jslib to be merged + updated.

File: apps/desktop/src/app/services/services.module.ts [hunk 1]
```diff
@@ -38,10 +38,10 @@ import { ElectronRendererStorageService } from "jslib-electron/services/electron
 
 import { Account } from "../../models/account";
 import { I18nService } from "../../services/i18n.service";
-import { LoginGuardService } from "../../services/loginGuard.service";
```
Comment (eliykat) [inline apps/desktop/src/app/services/services.module.ts:41]: See linting feedback for this file:

```
apps/desktop/src/app/services/services.module.ts#L8
'WINDOW' is defined but never used
```

File: apps/desktop/src/app/vault/groupings.component.ts [hunk 2]
```diff
@@ -1,20 +0,0 @@
-import { Component } from "@angular/core";
```
Comment (eliykat) [inline apps/desktop/src/app/vault/groupings.component.ts:1]: The html template for this component also needs to be deleted.

**Review by eliykat — CHANGES_REQUESTED** [review]

File: apps/desktop/src/app/services/services.module.ts (hunk 1 above)
Comment (Hinton) [reply apps/desktop/src/app/services/services.module.ts:41]: I believe that's from `master`, and it should be fixed in #2643.

**Review by addisonbeck — APPROVED** [review]
LGTM

Action errors are almost certainly due to the recent monorepo move for desktop, and should be cleared up soon

**Pull request merged**